files/bootstrap.sh

```diff
@@ -117,9 +122,13 @@ INSTANCE_TYPE=$(curl -s http://169.254.169.254/latest/meta-data/instance-type)
 
 if [[ "$USE_MAX_PODS" = "true" ]]; then
     MAX_PODS_FILE="/etc/eks/eni-max-pods.txt"
+    set +o pipefail
     MAX_PODS=$(grep $INSTANCE_TYPE $MAX_PODS_FILE | awk '{print $2}')
+    set -o pipefail
     if [[ -n "$MAX_PODS" ]]; then
         echo "$(jq .maxPods=$MAX_PODS $KUBELET_CONFIG)" > $KUBELET_CONFIG
+    else
+        echo "No entry for $INSTANCE_TYPE in $MAX_PODS_FILE. Not setting max pods for kubelet"
```

Comment:
So from now on, an instance type in `/etc/eks/eni-max-pods.txt`, but with no `MAX_PODS` will work and just not set a `maxPods` limit? 

And missing instances will print `"Exited with error on line 126"`?

Reply:
No, we'll just output the message when an instance type isn't available. 

The trap will just be there to help debug any other future error case